### PR TITLE
TG-119: buttons style

### PR DESCRIPTION
**Pull request by ilcosme**
Request to merge to master branch.

I changed the styling of the button shared component and also the style of the button implemented on the table shared component. In adittion both color and font style of the table header was changed. All changes were made accordingly to QA indications.

### CODE REVIEW

**Review by nicolascostanza — APPROVED** [review]

**Comment by karu-ams** [comment]
## TEST RESULTS
_Tested locally on https://alfonso-trackgenix-r9dij3maw-basp-m2022.vercel.app/_

**Test state: Changes are requested ⚠️** 
1. When the user click the button, the text turns red. I like this, but I suggest another color because of the contrast between the blue and red colors.
chrome-capture-2022-5-15

2. When I click the button, later it turns grey, unless I clicked again outside the button. This shouldn't happening because it seems to be disabled and might confuse the user.
chrome-capture-2022-5-15 (1)

File: src/Components/Shared/Button/Button.module.css [hunk 1]
```diff
@@ -1,34 +1,27 @@
...
   display: flex;
-  justify-content: center;
-  width: 220px;
+  justify-content: space-evenly;
+  width: 250px;
   height: 60px;
   padding: 10px;
   font-weight: bold;
   color: white;
+  text-transform: uppercase;
```
Comment (martinalbrnz) [inline src/Components/Shared/Button/Button.module.css:9]: like it

**Review by karu-ams — APPROVED** [review]
## TEST RESULTS ❗ 
_Tested locally on https://alfonso-trackgenix-p19s4npjw-basp-m2022.vercel.app/_

**Test: PASS** ✔️ _READY TO MERGE_

💃 💃 💃 💃 💃 💃 💃 💃 💃 💃 💃 💃 💃

**Review by martinalbrnz — APPROVED** [review]
Epic!

**Pull request merged**